plugins/README.md

```diff
@@ -0,0 +1,37 @@
+# Git Theta Plug-ins
```

Comment:
Don't need to address now but we should make sure to include this in the docs, not just a readme

Reply:
Make sense